### PR TITLE
ta_tc/media: Support media_utc when GMOCK is off

### PR DESCRIPTION
**Pull request by junmin-kim**
If CONFIG_GMOCK is not defined, use macro of tc_common.
More testcases will be included

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [9ee20f591b019c574f8dbc464625cbe7cca14754] - Code Rule Check Result:
apps/examples/testcase/ta_tc/media/utc/gmock/utc_media_common.h:28: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
apps/examples/testcase/ta_tc/media/utc/gmock/utc_media_common.h:30: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:WxO)
apps/examples/testcase/ta_tc/media/utc/gmock/utc_media_common.h:30: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
apps/examples/testcase/ta_tc/media/utc/gmock/utc_media_common.h:30: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
apps/examples/testcase/ta_tc/media/utc/gmock/utc_media_common.h:30: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)

**Comment by seinfra** [comment]
Target : [9ee20f591b019c574f8dbc464625cbe7cca14754] - Code Rule Check (C++) OK.

File: apps/examples/testcase/ta_tc/media/utc/utc_media_fileoutputdatasource.cpp [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /****************************************************************************
  *
- * Copyright 2018 Samsung Electronics All Rights Reserved.
+ * Copyright 2016 Samsung Electronics All Rights Reserved.
```
Comment (Taejun-Kwon) [inline apps/examples/testcase/ta_tc/media/utc/utc_media_fileoutputdatasource.cpp]: 18

File: apps/examples/testcase/ta_tc/media/utc/utc_media_mediaplayer.cpp [hunk 2]
```diff
@@ -0,0 +1,48 @@
+/****************************************************************************
+ *
+ * Copyright 2016 Samsung Electronics All Rights Reserved.
```
Comment (Taejun-Kwon) [inline apps/examples/testcase/ta_tc/media/utc/utc_media_mediaplayer.cpp]: 18

File: apps/examples/testcase/ta_tc/media/utc/utc_media_mediarecorder.cpp [hunk 3]
```diff
@@ -0,0 +1,48 @@
+/****************************************************************************
+ *
+ * Copyright 2016 Samsung Electronics All Rights Reserved.
```
Comment (Taejun-Kwon) [inline apps/examples/testcase/ta_tc/media/utc/utc_media_mediarecorder.cpp]: 18

**Comment by junmin-kim** [comment]
@Taejun-Kwon Fixed you commented

**Comment by seinfra** [comment]
Target : [1517a2a06f4f19410c0deb3699359a657ca966f9] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [1517a2a06f4f19410c0deb3699359a657ca966f9] - Code Rule Check Result:
apps/examples/testcase/ta_tc/media/utc/gmock/utc_media_common.h:28: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
apps/examples/testcase/ta_tc/media/utc/gmock/utc_media_common.h:30: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:WxO)
apps/examples/testcase/ta_tc/media/utc/gmock/utc_media_common.h:30: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
apps/examples/testcase/ta_tc/media/utc/gmock/utc_media_common.h:30: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
apps/examples/testcase/ta_tc/media/utc/gmock/utc_media_common.h:30: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)

File: apps/examples/testcase/ta_tc/media/utc/utc_media_fileinputdatasource.cpp [hunk 4]
```diff
@@ -16,105 +16,20 @@
  *
  ****************************************************************************/
 
-#include <stdio.h>
-#include <string.h>
+#include <tinyara/config.h>
```
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/media/utc/utc_media_fileinputdatasource.cpp]: Here is no CONFIG_XX. This looks not needs.

File: apps/examples/testcase/ta_tc/media/utc/utc_media_fileoutputdatasource.cpp [hunk 5]
```diff
@@ -16,116 +16,20 @@
  *
  ****************************************************************************/
 
-#include "utc_media_common.h"
+#include <tinyara/config.h>
```
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/media/utc/utc_media_fileoutputdatasource.cpp]: same as previous

File: apps/examples/testcase/ta_tc/media/utc/utc_media_mediaplayer.cpp [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#include <tinyara/config.h>
```
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/media/utc/utc_media_mediaplayer.cpp]: same

**Comment by junmin-kim** [comment]
@sunghan-chang I fixed you commented

**Comment by seinfra** [comment]
Target : [4012be4e50e4d4864a3020dc0245101e6650259e] - Code Rule Check Result:
apps/examples/testcase/ta_tc/media/utc/gmock/utc_media_common.h:28: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
apps/examples/testcase/ta_tc/media/utc/gmock/utc_media_common.h:30: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:WxO)
apps/examples/testcase/ta_tc/media/utc/gmock/utc_media_common.h:30: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
apps/examples/testcase/ta_tc/media/utc/gmock/utc_media_common.h:30: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
apps/examples/testcase/ta_tc/media/utc/gmock/utc_media_common.h:30: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)

**Comment by seinfra** [comment]
Target : [4012be4e50e4d4864a3020dc0245101e6650259e] - Code Rule Check (C++) OK.

**Pull request merged**